src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -91,7 +98,7 @@ public void updateFilteredListToShowAll() {
 
     @Override
     public void updateFilteredTaskList(Set<String> keywords){
```

Comment:
This is from find command. "Delete branch" should be merged with "find branch" before merged to master. See who created the prs. Im assuming this is what you two agreed upon.